alternator/streams.cc

```diff
@@ -884,8 +885,17 @@ future<executor::request_return_type> executor::get_records(client_state& client
     auto partition_slice = query::partition_slice(
         std::move(bounds)
         , {}, std::move(regular_columns), selection->get_query_options());
+    auto& opts = base->cdc_options();
```

Comment:
That's an unrelated optimization, no? Or at least not directly related to EOR?

Reply:
In a sense maybe